src/main/webapp/js/navigation-loader.js

```diff
@@ -34,6 +34,9 @@ function addLoginOrLogoutLinkToNavigation() {
           navigationElement.appendChild(createListItem(createLink(
               '/user-page.html?user=' + loginStatus.username, 'Your Page')));
 
+          navigationElement.appendChild(
+          	  createListItem(createLink('/feed.html', 'Public Feed')));
+
```

Comment:
Right now, the public feed page _link_ is only viewable if the user is logged in, but the feed page can be accessed regardless. This may be something to be fixed in the future - should a user who is not logged in be able to view the feed or not?

Reply:
I think it depends on what kind of user base we'll have, and what kind of content might be posted on it (where users may or may not want their posts to only be seen by friends i.e. people logged in). For now, since individual user profiles are visible using the link without having logged in, I'll make the public feed visible without being logged in, for consistency. We can adjust this in the future if need be.

Reply:
Sounds good!